Project-3/Jvonderh Project 3

```diff
@@ -0,0 +1,220 @@
+  Object Aggregate
+  (h) Lake
+  Object Aggregate
+```
+
+7. True or False; explain your answers:
+```
+  (a) An instance of Material Entity can have an instance of Immaterial Entity as part.
+  True. A material entity has some portion of matter as proper or improper continuant part. These parts don't need to be material. For example thoughts are immaterial but can be understood as a part of the material person. 
+  (b) An instance of Immaterial Entity can have an instance of Material Entity as part.
+  False. An immaterial entity is immaterial, so it may be concretized in a material entity or otherwise but may not have it as part. For example, a thought may be an immaterial entity which has a material entity as a subject, but that subject is either not material (i.e is a representation of a material thing) or is not a part of that thought.
+  (c) An organization may have another organization as part.
+  True. Organizations break off separate organizations to perform different functions all of the time. A charity organization for finding housing for the homeless might break off local and regional organizations to better serve people at various locations for example.
+  (d) An organization may have no members as part. 
+  True. Organizations can continue their existence even after losing all members for a time. For example, the Soviet Union continued to exist for a time even after all member states left the organization. It's continuation, then, depended not on its members but instead on other factors like the organizational structure and rules.
+  (e) Any site is partially bounded by some instance of Material Entity.
+  True. All examples of sites used in the BFO Protege consist of gaps between material entities. Gaps such as this, while separate from their surrounding material entities, will be partially defined by those material entities. For example, the Grand Canyon is specifically a gap defined by surrounding cliffs.
+  (f) A book placed under the leg of a wobbly table has acquired a new function. 
+  False. The book may have been primarily used to read or write in, but it was always a function of it that it could take up space and act as a solid base for another object. Function is not limited by its purpose of creation, and instead includes its ability to hold up a table or any other such object by virtue of its physical make-up.
+  (g) A glass vase cushioned with packing tape for all time, has the disposition to break.
+  True. Being cushioned with packing tape does not change the dispositions of the vase itself. 
+  (h) Spacetime is a class in BFO.
+  False. The spatiotemporal region covers this, but this itself is not a class.
+  (i) The continuant fiat boundary class of BFO is closed, meaning, there are no subclasses beyond those identified presently in BFO. 
+  False. We could construct any number of subclasses of zero-, one-, or two-dimensional continuant fiat boundaries, which would be subclasses of continuant fiat boundary.
```

Comment:
A better explanation comes from its entry in BFO file (you can look up it in Protege):

"Continuant fiat boundary doesn't have a closure axiom because the subclasses don't necessarily exhaust all possibilities. An example would be the mereological sum of two-dimensional continuant fiat boundary and a one dimensional continuant fiat boundary that doesn't overlap it".

Reply:
Fair on this. I am keeping the explanation as is, as I can understand what I have written here and explain it at least somewhat. I could not say the same for the entry you mention there, as I am uncertain about closure axioms and mereological sums.